src/common/pure/RateInfo/index.tsx

```diff
@@ -150,8 +152,9 @@ export function RateInfo({
   }, [currentIsInverted, activeRate])
 
   const fiatAmount = useMemo(() => {
+    if (noFiat) return null
```

Comment:
Better to double-check with Michel, maybe we should be consistent with this

Reply:
Didn't get this one.
What are you proposing?